src/rebootstrap-raft/main.go

```diff
@@ -70,10 +70,15 @@ func (c *rebootstrapCommand) Info() *cmd.Info {
 
 // SetFlags is part of cmd.Command.
 func (c *rebootstrapCommand) SetFlags(f *gnuflag.FlagSet) {
+	_, asSnap := os.LookupEnv("SNAP")
+	pathPrefix := ""
```

Comment:
What about doing it slightly differently:
```suggestion
	defaultJujuDir := "/var/lib/juju"
	if _, asSnap := os.LookupEnv("SNAP"); asSnap {
	  defaultJujuDir = "/var/lib/hostfs/var/lib/juju"
	}
```
I think that would be clearer to understand what you're looking for.
I would also consider a validation that if "asSnap" is true, but a user is going to supply `rebootstrap-raft --juju-dir /var/lib/juju` because that looks correct on their system.

I wonder if we actually want to do something later, rather than changing the default.